src/lib.rs

```diff
@@ -800,6 +1067,56 @@ macro_rules! __pin_project_internal {
             $($variant)*
         }
     };
+
+    (@struct=>make_proj_replace_method;
+        [$proj_vis:vis]
+        [$proj_ty_ident:ident][$_proj_ty_ident:ident]
+        [$method_ident:ident $get_method:ident $($mut:ident)?]
```

Comment:
`$method_ident`/`$get_method`/`$mut` will always be `project_replace`/`get_unchecked_mut`/`mut` , right? If so, it would be better to replace them and remove this line.